test/unit/trigger-method.spec.js

```diff
@@ -141,8 +141,8 @@ describe('trigger event and method name', function() {
       this.childView.$el.click();
     });
 
-    it('should fire the event method', function() {
-      expect(this.onChildviewFooClickStub).to.have.been.called;
+    it('should fire the event method once', function() {
+      expect(this.onChildviewFooClickStub).to.have.been.calledOnce;
```

Comment:
this test was modified in the ES6 module switch, but somewhere along the way whatever was causing the problem was fixed. :tada: